.gitlab-ci.yml

```diff
@@ -751,10 +763,13 @@ agent_deb-x64-a6:
     - source /root/.bashrc && conda activate $CONDA_ENV
     - inv -e deps --no-checks --verbose --dep-vendor-only
     - export RELEASE_VERSION=$RELEASE_VERSION_6
-  <<: *skip_when_unwanted_on_6
   <<: *agent_build_common_deb
 
 agent_deb-x64-a7:
+  rules:
+    - <<: *if_not_version_7
+      when: never
+    - when: on_success
```

Comment:
Since `rules` default to `when: never` when they're not matched, would this be equivalent to:
```
rules:
  - <<: *if_version_7
```
?

If it does, we could simplify the rules sections a lot, given the number of times this pattern appears.